### DISCUSSION TITLE
Cheating, fraud

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#791] [Q&A] [answered]
Good afternoon. I would like to know how safe it is (how much fraud prevention is) when taking exams through SEB with manual configuration of Moodle 4.3 on students' devices (Mac, iPhone and Windows).
At the moment we note that it is possible to log in with the same user from another device, a fact that implies that the original user must, when he want to continue with his exam, log in again, but this does not remove him from the SEB or blocks the exam. From already thank you very much

In addition to this possibility, I request you to inform me if the SEB, running from Moodle 4.3, with manual configuration with exit password, allows preventing remote access from other devices, when students use Mac, iPhone or Windows.
Thank you!!!

**Comment by mlohnen** [comment, marked as answer]
I think that it is not something for SEB to prevent this.
This is the responsibilty of the system with which you take the exam.
Solutions are only one session per user or using 2FA to login.

**Reply by Rapacioli** [reply]
Thank you very much, How could 2FA login prevent fraud? We would take in-person exams, in the classroom, with the students' devices

**Reply by mlohnen** [reply]
Well you could let the student logon with 2FA via their smartphone and after that put the smartphone away.
Nobody else can login on that account since they would need the smartphone for 2FA.

**Reply by Rapacioli** [reply]
Very clever, thank you very much!

**Reply by dbuechel** [reply]
> I think that it is not something for SEB to prevent this. This is the responsibilty of the system with which you take the exam. Solutions are only one session per user or using 2FA to login.

That is exactly correct. However, when using SEB-Server (see e.g. https://seb-server.readthedocs.io/en/latest/monitoring.html), then you can actually detect multiple logins resp. sessions and react accordingly, no matter whether the LMS or web application itself safeguards against it (at least during an exam).

**Reply by danschlet** [reply]
@dbuechel good point, but it is important to note that this is only the case for a Moodle quiz if it is set up with the new [Moodle SEB Server integration](https://github.com/ethz-let/moodle-quizaccess_sebserver), which requires the [according plugin](https://github.com/ethz-let/moodle-quizaccess_sebserver) to be installed. Only then the SEB Server generated SEB Server BrowserExamKey is used. Otherwise, if not using this new integration and only the standard check with the Moodle-generated ConfigKey, students with access to the Moodle course can just log in directly there, without using SEB Server (which exam supporters/proctors might realize or they might not).

**Reply by Rapacioli** [reply]
@dbuechel @danschlet  Thank you so much! I'm going to contact the Moodle administrators to ask about the configuration we have.
Thank you!

**Reply by Rapacioli** [reply]
@danschlet @dbuechel Good afternoon! We were finally able to install the seb server and we are carrying out configuration tests. We use Moodle 4.3. The problem I currently have is that I can't get the students to open the exam. It just says that they should open the SEB before taking the exam but I don't understand how to direct them to the exam. Could you help me? Thank you so much

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/94466010/3d99cf6d-c473-402d-83c5-4c312f615a6e)

**Reply by dbuechel** [reply]
You'll need to use the client configuration for SEB Server, see also https://seb-server.readthedocs.io/en/latest/configurations.html. Please ask any further questions related to SEB Server in its repository: https://github.com/SafeExamBrowser/seb-server.
